test/test_turbulence_closures.jl

```diff
@@ -129,6 +130,36 @@ function time_step_with_tupled_closure(FT, arch)
     return true
 end
 
+function time_step_with_catke_closure(arch)
+    closure = CATKEVerticalDiffusivity()
+
+    model = NonhydrostaticModel(closure=closure,
+                                grid=RectilinearGrid(arch, size=(1, 1, 1), extent=(1, 2, 3)),
+                                buoyancy = BuoyancyTracer(),
+                                tracers = (:b, :e))
+
+    @test model.tracers.e.boundary_conditions.top.condition !== nothing
+
+    time_step!(model, 1, euler=true)
+
+    return true
+end
+
+function time_step_with_tupled_catke_closure(arch)
+    closure_tuple = (CATKEVerticalDiffusivity(), AnisotropicDiffusivity())
+
+    model = NonhydrostaticModel(closure=closure_tuple,
+                                grid=RectilinearGrid(arch, size=(1, 1, 1), extent=(1, 2, 3)),
+                                buoyancy = BuoyancyTracer(),
+                                tracers = (:b, :e))
+
+    @test model.tracers.e.boundary_conditions.top.condition !== nothing
+
+    time_step!(model, 1, euler=true)
```

Comment:
```suggestion
    time_step!(model, 1)
```

I don't think it matters whether `euler=true` here, and its better not to include it if we decide to stop supporting it someday.